.circleci/config.yml

```diff
@@ -65,8 +69,22 @@ jobs:
       MINIKUBE_WANTUPDATENOTIFICATION: false
       MINIKUBE_WANTREPORTERRORPROMPT: false
       SKAFFOLD_VERSION: v0.18.0
+      GO111MODULE: "on"
+      GOVERSION: 1.12.1
```

Comment:
If we're doing 1.12, why not 1.12.5?

Reply:
Also, isn't golang already installed from the docker image?

Reply:
It installs an old version of golang, that is why I had to install the golang version here. 

No problem on using 1.12.5 ...